tests/e2e/pageobjects/ide/plugins/GitPlugin.ts

```diff
@@ -26,7 +26,7 @@ export class GitPlugin {
         @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper,
         @inject(CLASSES.DialogWindow) private readonly dialogWindow: DialogWindow,
         @inject(CLASSES.TopMenu) private readonly topMenu: TopMenu,
-        @inject(CLASSES.QuickOpenContainer) private readonly quickOpenContainer: QuickOpenContainer
+        @inject(CLASSES.QuickOpenContainer) private readonly quickOpenContainer: QuickOpenContainer,
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.

Reply:
fixed with the linter code change